SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -673,7 +673,7 @@ void ActorSystem::ComponentUpdated(const Worker_EntityId EntityId, const Worker_
 		EventTracer->TraceEvent(COMPONENT_UPDATE_EVENT_NAME, "", Causes, CauseSpanIds.Num(),
 								[Object, TargetObject, EntityId, ComponentId](FSpatialTraceEventDataBuilder& EventBuilder) {
 									EventBuilder.AddObject(Object);
-									EventBuilder.AddObject(TargetObject, "TargetObject");
+									EventBuilder.AddObject(TargetObject, "target_object");
```

Comment:
Interested to know what prompted this change. I prefer how it looks tbh but just wondering,

Reply:
Ah there is a push to get the naming consistent across gdk/worker/runtime, I guess the data looks a bit more consistent between layers!